Core/Lib/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lens.Core.Lib.Extensions
+{
+    public static class DateTimeExtensions
+    {
+
+        /// <summary>
+        /// Gets the quarter of the year
+        /// </summary>
+        /// <param name="value"></param>
+        /// <returns></returns>
```

Comment:
[minor] Remove unused tags from comments (like <returns> or <param>). See the bellow comments on the methods, too.

Reply:
Fixed